tools/type_whisperer/typedb_gen.py

```diff
@@ -143,7 +147,7 @@ def NextVersionUpgrade(type_name, type_map, next_version_upgrade_memo, visited=N
       type_desc.qualified_package
       for type_name, type_desc in type_map.items()
       if NextVersionUpgrade(type_name, type_map, next_version_upgrade_memo)
-  ])
+  ]).union(set(['envoy.config.retry.previous_priorities', 'envoy.config.cluster.redis']))
```

Comment:
Why are these 2 special cased here but there are more in TYPE_UPGRADE_REGEXES?

Reply:
The other versionless packages had deprecated fields or other deps that forced them to be upgraded. These didn't, but we do it explicitly to leave behind the versionless world.